src/test/resources/application-test.yml

```diff
@@ -11,3 +13,5 @@ spring:
         static: eu-west-2
       sqs:
         enabled: false
+    compatibility-verifier:
+      enabled: false # TODO: For unknown reasons there is currently a flagged incompatibility between Spring Cloud 2022.0.4 and Spring Boot 3.1.x.
```

Comment:
I couldn't figure out why this was needed, the compatibility matrix suggests these versions are compatible and they work in tis-trainee-credentials with similar versions. If anyone can figure it out it would be appreciated

Reply:
We are not the only ones with this issue, e.g. https://spring.io/blog/2023/07/28/spring-cloud-2022-0-4-aka-kilburn-is-available ` I can confirm that Spring 3.1.x is not compatible with this release of Spring Cloud. I was also under the impression that 2022.0.4 would be based on Spring Boot 3.1.x :(`
I am not sure why its working ok with tis-trainee-credentials though...